ui/src/components/search-result.tsx

```diff
@@ -1,149 +1,177 @@
+/* 
+This file contains the code for the <SearchResult/> Component.
+See "SearchResultProps" on this document for the the properties this component
+uses.
+*/
 
 import React from 'react';
 
+// If not working, run "npm install react-icons"
 import { FaConfluence, FaSlack, FaGoogleDrive } from "react-icons/fa";
 
-import BlueFolder from '../assets/images/blue-folder.svg';
+// This imports the full-color icons
 import Slack from '../assets/images/slack.svg';
 import Confluence from '../assets/images/confluence.svg';
-import GoogleDrive from '../assets/images/google-drive.svg';
+import GoogleDrive from '../assets/images/google-drive.svg'
 
-export interface TextPart{
-    content: string
-    bold: boolean
+export interface TextPart {
+  content: string
+  bold: boolean
 }
-  
+
 export enum ResultType {
-    Docment = "document",
-    Message = "message",
-    Comment = "comment",
-    Person = "person"
-}
-  
-export enum Platform {
-    Confluence = "confluence",
-    Slack = "slack",
-    Drive = "google_drive"
+  Docment = "document",
+  Message = "message",
+  Comment = "comment",
+  Person = "person"
 }
 
-export enum PlatformDisplayName {
-    Confluence = "Confluence",
-    Slack = "Slack",
-    Drive = "Google Drive"
+export enum Platform {
+  Confluence = "confluence",
+  Slack = "slack",
+  GoogleDrive = "google drive"
 }
 
+// These are the attributes of a <SearchResult/> react component.
 export interface SearchResultProps {
-    title: string 
-    author: string
-    author_image_url: string
-    author_image_data: string
-    time: string
-    content: TextPart[]
-    score: number
-    location: string
-    platform: string 
-    type: ResultType
-    url: string
+  title: string
+  author: string
+  author_image_url: string
+  author_image_data: string
+  time: string
+  content: TextPart[]
+  score: number
+  location: string
+  platform: string
+  type: ResultType
+  url: string
 }
 
 export const SearchResult = (props: SearchResultProps) => {
-    return (
```

Comment:
it shows like you've replaced the entire functions because of 3 -> 2 spaces, undo so I can see exactly what you've changed.